src/widgets/types.ts

```diff
@@ -117,3 +112,100 @@ export type PatientNotes = {
   };
   encounterProviders: { provider: { person: { display: string } } };
 };
+
+export interface AllergyData {
+  allergen: {
+    allergenType: string;
+    codedAllergen: {
+      answers: [];
+      attrributes: [];
+      conceptClass: DisplayMetadata;
+      display: string;
+      links: Links;
+      mappings: DisplayMetadata[];
+      name: {
+        conceptNameType: string;
+        display: string;
+        locale: string;
+        name: string;
+        uuid: string;
+      };
+      names: DisplayMetadata[];
+      setMembers: [];
+      uuid: string;
+    };
+  };
+  auditInfo: {
+    changedBy: DisplayMetadata;
+    creator: DisplayMetadata;
+    dateCreated: string;
+    dateChanged: string;
+  };
+  comment: string;
+  display: string;
+  links: Links;
+  reactions: [
+    {
+      reaction: AllergicReaction;
+    }
+  ];
+  severity: {
+    name: {
+      conceptNameType: string;
+      display: string;
+      locale: string;
+      name: string;
+      uuid: string;
+    };
+    names: DisplayMetadata[];
+    uuid: string;
+  };
+}
+
+export type Allergen = {
+  answers: [];
+  attributes: [];
+  conceptClass: DisplayMetadata;
+  dataType: DisplayMetadata;
+  descriptions: [];
+  display: string;
+  links: Links;
+  mappings: Array<DisplayMetadata>;
+  name: {
+    display: string;
+    links: Links;
+    uuid: string;
+    conceptTypeName: string | null;
+    locale: string;
+    localePreferred: boolean;
+    name: string;
+    resourceVersion: string;
+  };
+  names: DisplayMetadata[];
+  setMembers: [];
```

Comment:
See all the other sources (I only remark this once per file); this is I guess not what we want for typing. Can you be more specific? Thanks!